src/io/flutter/preview/PreviewView.java

```diff
@@ -179,13 +179,13 @@ public void onFailure(RenderProblemKind kind) {
           switch (kind) {
             case EXCEPTION:
             case NO_TEMPORARY_DIRECTORY:
-              previewArea.clear("There was an exception during rendering");
+              previewArea.clear("Encountered an exception during rendering");
               break;
             case NO_WIDGET:
               previewArea.clear(PreviewArea.NO_WIDGET_MESSAGE);
               break;
             case NOT_RENDERABLE_WIDGET:
-              previewArea.clear("The selection does not correspond to a renderable widget");
+              previewArea.clear(PreviewArea.NO_WIDGET_MESSAGE);
```

Comment:
We lose distinction between "not widget" and "not renderable widget".
Is this intentional?

Reply:
It was not - I'll update.